packages/react/src/main-nav/__snapshots__/MainNav.test.tsx.snap

```diff
@@ -27,30 +27,23 @@ exports[`MainNav renders correctly 1`] = `
           >
             <svg
               aria-hidden="true"
-              fill="currentcolor"
+              class="css-9s3cfn-Icon"
+              clip-rule="evenodd"
               focusable="false"
               height="24"
+              role="img"
```

Comment:
Is this role appropriate if it is aria hidden?

Reply:
Hmm, that's a good point – probably not. We probably also don't need `focusable` as we don't need to support IE. 

But lets make that change later globally by updating the `createIcon` function.